.github/workflows/cpaas.yml

```diff
@@ -82,18 +82,13 @@ jobs:
           username: ${{ secrets.QUAY_STACKROX_IO_RW_USERNAME }}
           password: ${{ secrets.QUAY_STACKROX_IO_RW_PASSWORD }}
 
-  build-collector-full:
-    uses: ./.github/workflows/collector-full.yml
-    with:
-      collector-tag: ${{ needs.init.outputs.collector-tag }}-cpaas
-      arch-drivers-bucket: ${{ needs.init.outputs.cpaas-drivers-bucket }}
-      skip-built-drivers: true
-      archs: amd64 ppc64le s390x
-    secrets: inherit
-    needs:
-    - init
-    - build-collector-slim
-    - sync-drivers-and-support-packages
+      - name: Retag and push stackrox-io -amd64-slim
+        uses: stackrox/actions/images/retag-and-push@v1
+        with:
+          src-image: brew.registry.redhat.io/rh-osbs/rhacs-collector-slim-rhel8:0.1.0
+          dst-image: quay.io/rhacs-eng/collector:${{ needs.init.outputs.collector-tag }}-cpaas-${{ matrix.arch }}-slim
+          username: ${{ secrets.QUAY_RHACS_ENG_RW_USERNAME }}
+          password: ${{ secrets.QUAY_RHACS_ENG_RW_PASSWORD }}
```

Comment:
I noticed you had to add the `collector-test-tag` to some workflows because there is no multiarch manifest for this image, wonder if it'd be easier to retag and push without the `-slim` part, then simply use the `create-multiarch-manifest` action instead.

Reply:
Also, what's the difference between this step and the one before? isn't `collector-image` the same as `quay.io/rhacs-eng/collector:${{ collector-tag }}`?

Reply:
I'm not sure I understand what you mean about using the create-multiarch-manifest action. We need to push -slim versions of the images before we're able to then create a -slim manifest.

As for the duplication with the previous step... yes that is odd. Not sure what I was thinking here, i'll double check.